framework/src/utils/MooseMeshUtils.C

```diff
@@ -144,4 +144,24 @@ getSubdomainIDs(const libMesh::MeshBase & mesh, const std::vector<SubdomainName>
 
   return ids;
 }
+
+bool
+isBoundaryID(const BoundaryName & boundary_name)
+{
+  std::istringstream ss(boundary_name);
+  BoundaryID id;
+  return !(!(ss >> id));
```

Comment:
so if the boundary name is 5 but it s boundary id is 4 is it going to be an issue?

Reply:
Dunno - this was used in MOOSE previously, so I trusted it.

Reply:
The function name is not correct; should be something like, couldThisBoundaryNameBeABoundaryID. In truth, should this be improved to just check the Mesh like in getBoundaryID?